### ISSUE TITLE
Stucks when there are a lot of resources 100k +

### ISSUE THREAD

**Issue by vsvfourth** [tom-englert/ResXResourceManager#303]
-When programs start it tries to analyze all resx
-When there a thousands of resources plugin crashes
-New logic to analyze it per language 
-Setting which folder to analyze

**Comment by tom-englert** [comment]
Usually the problem is not to load all resx files, but to try to visualize them all at the same time.

Therefore you can already select only a subset of your projects in the left panel, or only specific language columns.
You may also unload some projects in your VisualStudio solution to reduce the amount of projects analyzed.
If you like to work only on specific folders, use the standalone version.

If you encounter crashes, provide proper dumps or steps and sample project to reproduce it.

**Comment by IoTFier** [comment]
Hi @tom-englert, how are you ? 

I am using the standalone version. 
My App resource file has 325 resources which I am trying to translate to 5-6 languages over last couple days as it limits the number of resources to be translated per daily quota.
Where is that information in the documentation? I couldn't fine one.

For 300-500 resource will Azure Translation F0 service be a viable option? 
Do you have any status / limitation info per translation service provider that are part of the latest Standalone?

I have been waiting 16 hours or so to handle this data limit, but getting frustrated as it never completes complete translation even one language at time.

Your guidance will be appreciated. 

Thanks

**Comment by tom-englert** [comment]
Sorry, RXRM only forwards the request, you have to check this with the provider you are using

**Comment by IoTFier** [comment]
Hi @tom-englert , 

I get that error immediately for each subsequent provider once I get the error from the 1st one.
Just wondering if the connection is switching to the next one when selected from the left tabs on the Translation tab.

Thank you

**Comment by tom-englert** [comment]
@IoTFier no, they just run in parallel, they are fully independent. BTW why are you commenting this on a totally unrelated issue?

**Comment by tom-englert** [comment]
@vsvfourth could you solve this by selecting only a subset?

**Comment by IoTFier** [comment]
> @vsvfourth could you solve this by selecting only a subset?

Yes, I tried that and it wasn't working.

**Comment by IoTFier** [comment]
> @IoTFier no, they just run in parallel, they are fully independent. BTW why are you commenting this on a totally unrelated issue?

Hi @tom-englert - I appended my issue in this to emphasize that even for smaller set of resources I was getting stuck. I guess I just fell for the subject line. More than happy to spin separate issue.
Besides, based on the UI - as a user I felt only the selected services is fired up in the background. If you are saying all are fired up and it is not clear which services is throwing that message.
Truly, thank you for your help in putting this capability for us.

**Comment by tom-englert** [comment]
> 
> 
> > @vsvfourth could you solve this by selecting only a subset?
> 
> Yes, I tried that and it wasn't working.

@IoTFier And why are you answering questions that are not addressed to you?

**Comment by tom-englert** [comment]
Closing this because it is totally screwed up now. If s.o. wants to continue on one of the threads, please create a new issue.
